src/components/Card/Card.module.scss

```
@@ -0,0 +1,58 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  padding: 8px 12px;
+  width: 155px;
+  position: relative;
+  background: #FFFFFF;
+  box-shadow: 0px 1px 8px rgba(0, 0, 0, 0.12);

```

здесь и где ещё найдётся
```suggestion
  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.12);
```

некрит, но глаз цепляется